pkgs/development/python-modules/django-extensions/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, django, six }:
+
+buildPythonPackage rec {
+  pname = "django-extensions";
+  version = "1.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "08rd9zswvjb9dixzyd3p3l3hw3wwhqkgyjvid65niybzjl1xdb5h";
+  };
+
+  doCheck = false;
```

Comment:
The test was failed at the time I create this PR, due to many dependencies are missing.

I fix this in next commit. Now pass all the tests.